android/app/src/main/java/updated/mysterium/vpn/model/filter/NodeType.kt

```diff
@@ -0,0 +1,16 @@
+package updated.mysterium.vpn.model.filter
+
+typealias FullNodeType = network.mysterium.proposal.NodeType
+
+enum class NodeType {
+    ALL, RESIDENTIAL, NON_RESIDENTIAL;
```

Comment:
Enum values should be on separate lines.

Reply:
Fix